core-bundle/assets/styles/jobs/menu.pcss

```
@@ -0,0 +1,49 @@
+[data-controller="contao--jobs"] {
+    #running-jobs {
+        display: none;
+    }
+    
```

```suggestion

    sup {
        pointer-events: none;

        &:empty {
            display: none;
        }
    }

```

We do not want to display the `sup` when there is no messages (and definitely do not want to make it cover the button 🤔)